arbor/morph/morphology.cpp

```diff
@@ -365,35 +328,6 @@ bool mextent::test_invariants(const morphology& m) const {
     // Too many branches?
     if (!empty() && cables_.back().branch>=m.num_branches()) return false;
```

Comment:
Is this worth an extra `test_invariants` variant?

Reply:
I think so, because it asserts that the extent is valid for a given morphology.